accessibility/index.html

```diff
@@ -0,0 +1,758 @@
+					</section>
+					<section>
+                        <p><img src="images/iOS03.png" border="0"> &nbsp; &nbsp; <img src="images/iOS04.png" border="0"></p>
+					</section>
+                    <section>
+                        <p><img src="images/iOS05.png" border="0"></p>
+					</section>
+				</section>
+
+
+				<!-- HOW TO ACTIVATE VOICEOVER FOR iOS END -->		
+				
+				<!-- HOW TO USE VOICEOVER FOR IOS START -->
+
+				<section>
+					<section class="fragments">
+                        <h2>Using <span class="gdi">VoiceOver</span><br/>Screen Reader on <span class="gdi">iPhone</span></h2>
+                        <p>Once VoiceOver is Activated</p>
+						<p class="fragment">Activate an Item on the Screen: Double Tap</p>
+						<p class="fragment">Next Item: Flick Right</p>
+						<p class="fragment">Previous Item: Flick Left</p>
+						<p class="fragment">Scroll Down/Up: Three Finger Up/Down</p>
+						<p class="fragment">Activate Rotor: Turn Clock-wise</p>
+						<p class="fragment">Next/Previous Rotor Item: Flick Down/Flick Up</p>
+					</section>
+                    <aside class="notes">
+				    Instructions for using VoiceOver on an OSX device once it has been enabled. - Demo and Group Activity
+                    </aside>
+				</section>
+				
+				<!-- HOW TO USE VOICEOVER FOR IOS END -->
+                        
+                <!-- HOW TO ACTIVATE VOICEOVER FOR OSX START -->
+                        
+                <section>
+						<h2>Activate <span class="gdi">VoiceOver</span><br/>Screen Reader - OSX</h2>
+                    <small><span class="gdi">Instructions</span> for activating VoiceOver for OSX.</small>
+                    <p><strong><span class="gdi">Activate</span> VoiceOver</strong> from <span class="gdi">System Preferences</span> > <span class="gdi">Accessibility</span> > <span class="gdi">Enable VoiceOver</span> checkbox</p>
+				 <aside class="notes">
+                        Instructions for enabling VoiceOver for OSX
+                    </aside>
+                </section>
+                        
+                <!-- HOW TO ACTIVATE VOICEOVER FOR OSX END -->
+                
+                <!-- USING VOICEOVER FOR OSX START -->
+                
+                <section>
+					<section class="fragments">
+                        <h2>Using <span class="gdi">VoiceOver</span><br/>Screen Reader for <span class="gdi">OSX</span></h2>
+                        <p>Once VoiceOver is Activated</p>
+						<p class="fragment">Tun on VoiceOver: command + F5</p>
+                        <p class="fragment">Use the VO Keys for many tasks = control + option</p>
+                    </section>
+                    <aside class="notes">
+				    Instructions for using VoiceOver on an OSX device once it has been enabled. - Demo and Group Activity
+                    </aside>
+				</section>
+                
+                <!-- USING VOICEOVER FOR OSX END -->
+                <section>
```

Comment:
We also commonly have Linux users. If this happens during class, I recommend suggesting that they pair with somebody who has an iOS or Mac OS X laptop